src/plugins/misc/commands/jumbo.command.ts

```diff
@@ -0,0 +1,52 @@
+import {ChatInputCommandInteraction, inlineCode, PermissionFlagsBits} from "discord.js";
+import {SlashCommandBuilder, SlashCommandScope} from "../../../builders/SlashCommandBuilder";
+import {useChatCommand} from "../../../hooks/useChatCommand";
+import axios from 'axios';
+
+const builder = new SlashCommandBuilder()
+    .setName("jumbo")
+    .setDescription("Make G'day blow up an image.")
+    .addStringOption(option => option.setName("emoji").setDescription("Emoji to jumbo size.").setRequired(true))
+    .setDefaultMemberPermissions(PermissionFlagsBits.ManageMessages)
+    .setEphemeral(true)
+    .setScope(SlashCommandScope.GLOBAL);
+
+useChatCommand(builder, async (interaction: ChatInputCommandInteraction) => {
+    let emoji = interaction.options.getString('emoji')?.trim();
+
+    if (!emoji || !interaction.channel) {
+        return "Invalid request.";
+    }
+
+    // Handle custom emojis
+    if (emoji?.startsWith("<") && emoji.endsWith(">")) {
+        const match = emoji.match(/\d{15,}/g);
+        if (!match) {
+            return "Invalid emoji format!";
+        }
+
+        const id = match[0];
+
+        let type = "png";
+        try {
+            const image = await axios.get(`https://cdn.discordapp.com/emojis/${id}.gif`);
```

Comment:
This is potentially the least efficient way of doing this, all animated emojis start with `<a:`, there is no need to make an http request to see if it's animated

Reply:
I see, made just an if statement to check for `<a:` at the start.